README.md

```diff
@@ -130,6 +130,28 @@ Changes the passphrase for given role keys file. The CLI supports reading
 both the existing and the new passphrase via the following environment
 variables - `TUF_{{ROLE}}_PASSPHRASE` and respectively `TUF_NEW_{{ROLE}}_PASSPHRASE`
 
+#### `tuf payload <metadata>`
+
+Outputs the metadata file for a role in a ready-to-sign (canonicalized) format.
```

Comment:
FYI sorry about the badgering about the canonicalization again but if https://github.com/theupdateframework/go-tuf/pull/247/files?file-filters%5B%5D=.go&show-viewed-files=true is merged first (trynna get that in...), then this won't hold true -- metadata to the store won't be written in canonical form ahead of time.

Currently the tests use `SignPayload` which will internally canonicalize -- do you think you could add a test demonstrating that Payload() is canonical? Maybe show Canonicalize is no-op to the output

Reply:
Ha, fair. Added [ee3e8e](https://github.com/theupdateframework/go-tuf/pull/214/commits/ee3e8ee3c70b0c35316338f53b1ab4afb66e396e) to check. (Turns out that `cjson.EncodeCanonical` is not idempotent!)